examples/Using_tool_required_for_customer_service.ipynb

```diff
@@ -0,0 +1,515 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "dd290eb8-ad4f-461d-b5c5-64c22fc9cc24",
+   "metadata": {},
+   "source": [
+    "# Using Tool Required for Customer Service\n",
+    "\n",
+    "The `ChatCompletion` endpoint now includes the ability to specify whether a tool **must** be called every time. This adds an element of determinism to how you build your wrapping application, as you can count on a tool being provided with every call. \n",
+    "\n",
+    "We'll demonstrate here how this can be useful for a contained flow like customer service, where having the ability to define specific exit points gives more control.\n",
+    "\n",
+    "The notebook concludes with a multi-turn evaluation, where we spin up a customer GPT to imitate our customer and test the LLM customer service agent we've set up."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
```

Comment:
can you run the cells all from the start to have them display as 1, 2, 3 ....?

Reply:
This is now done